blob/blob.go

```diff
@@ -522,7 +535,10 @@ func (b *Bucket) WriteAll(ctx context.Context, key string, p []byte, opts *Write
 //
 // The caller must call Close on the returned Writer, even if the write is
 // aborted.
-func (b *Bucket) NewWriter(ctx context.Context, key string, opts *WriterOptions) (*Writer, error) {
+func (b *Bucket) NewWriter(ctx context.Context, key string, opts *WriterOptions) (_ *Writer, err error) {
```

Comment:
The traces for `NewRangeReader` and `NewWriter` probably aren't that useful, as they don't usually make RPCs either. Could we start a trace here, and close it when the `Reader`/`Writer` is closed?